docs/root/configuration/http/http_filters/golang_filter.rst

```diff
@@ -13,8 +13,8 @@ See the `Envoy's Golang extension proposal documentation
 for more details on the filter's implementation.
 
 .. attention::
-  It is designed to work with the `GODEBUG=cgocheck=0` ENV variable, when running envoy with Go plugin enabled,
-  which disables the cgo pointer check.
+  The Envoy Golang filter is designed to be run with the `GODEBUG=cgocheck=0` environment variable set,
+  when running Envoy with Golang filter enabled, which disables the cgo pointer check.
```

Comment:
```suggestion
  The Envoy Golang filter is designed to be run with the `GODEBUG=cgocheck=0` environment variable set.
  
  This disables the cgo pointer check.
```

the sentence doesnt quite read correctly

Reply:
Thanks, fixed it with a force push, due to miss DSO in the previous commit.